industrial_msgs/package.xml

```diff
@@ -1,20 +1,25 @@
-<package>
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
   <name>industrial_msgs</name>
   <version>0.5.1</version>
   <description>
 	The industrial message package containes industrial specific messages 
 	definitions. This package is part of the ROS-Industrial program.
   </description>
+  <author>Shaun Edwards</author>
   <maintainer email="shaun.edwards@gmail.com">Shaun Edwards</maintainer>
   <license>BSD</license>
   <url type="website">http://ros.org/wiki/industrial_msg</url>
-  <author>Shaun Edwards</author>
 
   <buildtool_depend>catkin</buildtool_depend>
-  <build_depend>std_msgs</build_depend>
-  <build_depend>trajectory_msgs</build_depend>
+
+
   <build_depend>message_generation</build_depend>
```

Comment:
forgot to convert <build_depend>message_generation</build_depend> to format 2.

Reply:
@trentwall: could you be more specific? What do you believe it should be?

Reply:
<depend>roslaunch</depend>

Reply:
<depend>roslaunch</depend>

Reply:
I'm not sure I understand.

Reply:
depend roslaunch /depend

Reply:
just like the others

Reply:
but `roslaunch` is not the same package as `message_generation`. Package format 2 does not influence that.

Reply:
I talked to Trent, and he was confused about the new `<depend>` tag.  No changes are required.  This PR is correct as-is.